.changeset/rich-maps-hear.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-react': patch
+'@backstage/plugin-scaffolder': patch
+---
+
+Added a new `NextScaffolderRouter` which will eventually replace the exiting router
```

Comment:
Do we add some instructions for those who want to test early on?

Reply:
Not yet, until there's something that will allow scaffolding. but soon. 🔜